server/planning/commands/flag_expired_items_test.py

```diff
@@ -12,12 +12,14 @@
 
 from bson.objectid import ObjectId
 
+from planning.events.service import EventsAsyncService
+from planning.planning.service import PlanningAsyncService
 from superdesk import get_resource_service
 from superdesk.utc import utcnow
 
 from planning.tests import TestCase
 from planning.types import PlanningRelatedEventLink
-from .flag_expired_items import FlagExpiredItems
+from .flag_expired_items import flag_expired_items_handler
 
 now = utcnow()
 yesterday = now - timedelta(hours=48)
```

Comment:
It seems more like two days than yesterday 😉 
```suggestion
two_days_ago = now - timedelta(hours=48)
```